src/ansys/turbogrid/core/launcher/launcher.py

```diff
@@ -148,6 +149,8 @@ def launch_turbogrid(
         an available port is automatically selected.
     host : str, default: ``127.0.0.1 (this is the local host IP for windows and linux)``
         host for TurboGrid communications. The default is ``127.0.0.1, or the local Host``
+    log_filename_suffix : str, default: ""
```

Comment:
Capital "S" for suffix. Although not related to this change, the line above (151) should have a capital "H" for the first letter as well (and a small "h" in "local Host").